ios/src/TGMapViewController.h

```diff
@@ -12,6 +12,26 @@
 #import <UIKit/UIKit.h>
```

Comment:
Hmm.. Probably one more thing you can do here is to take this (c159079af9f57ff0e495a269a4cf850015693aca) commit out.

* `git rebase -i HEAD~12`
* remove the line mentioning this (c159079af9f57ff0e495a269a4cf850015693aca) commit.
* save
* to be sure you are all good, do a diff of this local branch with the remote.